src/tests/evil/unposix/eina_vasprintf_test.c

```diff
@@ -0,0 +1,63 @@
+#include "eina_stdio.h"
+#include <stdarg.h>
+#include <stdlib.h>
+#include <string.h>
+
+
+int vasprintf_wrap(char **ret, const char *fmt, int nargs, ...);
+int simple_test(void);
```

Comment:
No need to declare to then define in the same file - besides, that's exactly what the header (`eina_stdio.h`) is doing :)